tests/end_to_end/workflow/exclude_flow.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2020-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+
+from openfl.experimental.workflow.interface import FLSpec
+from openfl.experimental.workflow.placement import aggregator, collaborator
+
+log = logging.getLogger(__name__)
+
+class TestFlowExclude(FLSpec):
```

Comment:
Query: The original test case seems to have a functionality where the class attribute _exclude_error_list_ maintained the step in which the error occurred to help with troubleshooting. Is there a reason why that was removed ?